docs/test-prep.md

```diff
@@ -0,0 +1,406 @@
+# Preparing a test for MFTF
+
+This tutorial will guide you through the process of converting a raw functional test into a properly abstracted test file, ready for publishing.
```

Comment:
Here and further in the topic, try to avoid future tense.

Reply:
Done.